src/tests/algorithms/test_bias_triangles.py

```diff
@@ -0,0 +1,20 @@
+""" Test functionality to analyse bias triangles
+
+"""
```

Comment:
Same with the other file doc-stings...

Reply:
I'll do a quick scan for these type of docstring remarks. During my wok I saw a lot of docstring-findings. Incomplete descriptions of arguments, missing parameters etc. Maybe it's a good idea to do these kind of changes in a separate ticket because of the number of changes.